runtime/near-vm-runner/src/tests.rs

```diff
@@ -31,19 +31,24 @@ pub(crate) fn with_vm_variants(
     #[allow(unused)] cfg: &near_parameters::vm::Config,
     runner: impl Fn(VMKind) -> (),
 ) {
+    let run = move |kind| {
+        println!("running test with {kind:?}");
+        runner(kind)
+    };
```

Comment:
While debugging the tests I got bitten a few times by thinking there were issues with NearVm while the issues were actually in the tests. Hopefully with this we’ll waste fewer cycles in meaningless searches.